exporter/awsxrayexporter/awsxray.go

```diff
@@ -90,7 +90,8 @@ func newTraceExporter(
 			return err
 		},
 		exporterhelper.WithShutdown(func(context.Context) error {
-			return logger.Sync()
+			_ = logger.Sync()
```

Comment:
I don't think a component should flush or shutdown the logger. We should do that in the service in core since that is the part of the code that create it

Reply:
Is this the wrong logic here? If the logger should be handled upstream then this should just be a noop.

Reply:
I think so, but we should just open an issue to discuss this, and not block this PR.